core/src/main/java/tech/pegasys/web3signer/core/multikey/SignerLoader.java

```diff
@@ -41,33 +46,62 @@
 
     LOG.info("Loading signer configuration files from {}", configsDirectory);
     final AtomicLong configFilesHandled = new AtomicLong();
+
+    final List<Path> configFilesList = getConfigFilesList(configsDirectory);
+    LOG.info("Processing {} files from configuration directory", configFilesList.size());
+
+    // use custom ForkJoin with limited number of threads if default is greater than 10
+    Set<ArtifactSigner> artifactSigners = Collections.emptySet();
+    ForkJoinPool forkJoinPool = null;
+    try {
```

Comment:
can this be split into subfunctions? It feels a bit busy, and might be easier to digest if smaller chunks.

Reply:
refactored to sub methods.